aviary/models/N3CC/N3CC_FLOPS.csv

```diff
@@ -19,14 +19,12 @@ aircraft:crew_and_payload:flight_crew_mass_scaler,1,unitless
 aircraft:crew_and_payload:mass_per_passenger,165,lbm
 aircraft:crew_and_payload:misc_cargo,0,lbm
 aircraft:crew_and_payload:non_flight_crew_mass_scaler,1,unitless
+aircraft:crew_and_payload:num_business_class,20,unitless
```

Comment:
You shouldn't have to add these passenger values because they should be automatically setup based on the design values. I think you can remove these 4 lines of changes.

Reply:
I added these because they weren't getting set right, which must mean that a new bug has cropped up.